.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "Source/SubModules/FPGA Addon Speciality IO"]
```

Comment:
For now, this file should remain unchanged and we'll revisit it when fixing dependencies in a more organized way. 

Same goes for Source/SubModules/FPGA Addon Speciality IO